### PR TITLE
HDInsight Custom Policies

### PR DESCRIPTION
**Pull request by marvinbuss**
## Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. Added HDInsight Custom Policies

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

**Comment by marvinbuss** [comment]
@krnese I updated this branch to comply with the new policy definition model.

**Comment by marvinbuss** [comment]
@krnese This policy is ready for merging.

**Comment by marvinbuss** [comment]
Docs will be done later once we have finalized merging all datapolicies.

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Here's what's changed in Enterprise Scale:
 | Deny-Databricks-NoPublicIp | Deny public IPs for Databricks cluster | Databricks |  |  |  | Denies the deployment of workspaces that do not use the noPublicIp feature to host Databricks clusters without public IPs. |
 | Deny-Databricks-Sku | Deny non-premium Databricks sku | Databricks |  |  |  | Enforces the use of Premium Databricks workspaces to make sure appropriate security features are available including Databricks Access Controls, Credential Passthrough and SCIM provisioning for AAD. |
 | Deny-Databricks-VirtualNetwork | Deny Databricks workspaces without Vnet injection | Databricks |  |  |  | Enforces the use of vnet injection for Databricks workspaces. |
+| Deny-HdInsight-MinimalTlsVersion | Azure HDInsight clusters should use minimal TLS version | HDInsight |  |  |  | Enforces minimal TLS versions for HD Insight clusters. |
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:54]: Can we move these into a new October section please to align with the PR merge date 👍

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Hey @marvinbuss,

Just a small change here for the Wiki. Also did you complete any testing and provide any evidence for this change to the `dataPolicies.json` file? 

Thanks

JT

File: docs/wiki/Whats-new.md (hunk 1 above)
Comment (marvinbuss) [reply docs/wiki/Whats-new.md]: DONE!

**Comment by krowlandson** [comment]
- Closing as superseded by #1144

**Pull request closed without merging**